app/spdb/build.sbt

```diff
@@ -214,6 +214,7 @@ def swalker(version: Version) = AppConfig(
     "edu.gemini.auxfile.other.dest"              -> "/gemsoft/var/data/finder/GSqueue/Finders-Test/@SEMESTER@/@PROG_ID@",
     "edu.gemini.auxfile.root"                    -> "/Users/swalker/.auxfile",
     "edu.gemini.dataman.gsa.summit.host"         -> "cpofits-lv1new.cl.gemini.edu",
+    "edu.gemini.dbTools.tcs.ephemeris.directory" -> "/Users/swalker/.ephemeris",
```

Comment:
It isn't necessary unless you want to work with the ephemeris robot.  The spdb will start regardless and the ephemeris robot will even run if you trigger it, defaulting the directory to the cron job external data directory.